src/MacPawLogo/MacPawLogo.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, HTMLAttributes } from 'react';
+import cx from 'clsx';
+import { MacpawText, PawIcon } from '../Icons/jsx';
+
+export interface MacPawLogoProps extends HTMLAttributes<HTMLDivElement> {
+  classNames?: string;
+  disableRotate?: boolean;
+  pawProps?: { [key: string]: any};
+  textProps?: { [key: string]: any};
+  inline?: boolean;
+}
+
+const MacPawLogo: FC<React.PropsWithChildren<MacPawLogoProps>> = (props) => {
+  const { pawProps, textProps, classNames = '', disableRotate = false, inline } = props;
```

Comment:
- Looks like we should collect rest props using the spread operator `...rest` to pass something like an onClick handler in the future ? 
- Also, we should be able to use `Link` component or `a` tag instead of `div` tag to make the logo as a link to the website.
```suggestion
  const { pawProps, textProps, classNames = '', disableRotate = false, inline, ...rest } = props;
```